recipes/numba4jax/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "numba4jax" %}
+{% set version = "0.0.3" %}
+
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/numba4jax-{{ version }}.tar.gz
+  sha256: a88869874965bcaec2ce4deebe06a92dfa9102cc860702a2bad7aac8bec01ba6
+
+build:
+  number: 0
+  noarch: python
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script: 
+    - {{ PYTHON }} -m pip install --no-deps --ignore-installed --no-cache-dir -vvv .
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - setuptools >=45
+    - setuptools_scm >=3.4
+    - toml
+  run:
+    - cffi >=1.12
+    - jax >0.2.16,<0.2.25
+    - jaxlib >0.1.70
+    - numba >=0.52,<0.55
```

Comment:
Ditto for the bounds on numba and jax. Are these pins needed?

Reply:
Numba broke something I was relying on almost every .X release they have made in the last years. That's why I tend to bound it on my projects (so users don't have to worry about too recent versions). 

If this is not standard for conda users i can remove those upper bounds.

Reply:
I agree. numba has broken my projects as well. I'd prefer that we let users pin their envs and only exclude things we know are bad. For example, I have this in one of my recipes

```
- numba <0.41.0a|>=0.43.0,!=0.54.0  # bug in these numbas
```

I think that will work for you as well.

Reply:
Ok, i removed the bound and inserted the exception.

(Sorry for the noise,  I come from the julia-world where it is customary to upper bound everything (since it's easy to override them anyway) and they are modified after a new version is released upstream if needed, so that the whole ecosystem never breaks as long as everyone follows semantic versioning...)

Reply:
No worries at all! We also allow metadata overrides for similar reasons, but those only happen channel-wide and not per user. Generally, we don't enforce upper bounds in our pins so that software environments are maximally compatible as users install stuff.

Reply:
thank you for working with us!